pkg/mcs/resource_manager/server/manager.go

```diff
@@ -130,17 +130,18 @@ func (m *Manager) Init(ctx context.Context) {
 }
 
 // AddResourceGroup puts a resource group.
-func (m *Manager) AddResourceGroup(group *ResourceGroup) error {
+func (m *Manager) AddResourceGroup(grouppb *rmpb.ResourceGroup) error {
 	m.RLock()
-	_, ok := m.groups[group.Name]
+	_, ok := m.groups[grouppb.Name]
 	m.RUnlock()
 	if ok {
 		return errors.New("this group already exists")
 	}
-	err := group.CheckAndInit()
```

Comment:
`CheckAndInit` check whether `rg.RUSettings` is nil. If just remove it, resource manager server may be panic?

Reply:
I found `CheckAndInit` has redundant code and function with `FromProtoResourceGroup`, 
Do you think it's a good way to add check nil in `FromProtoResourceGroup` and replace `CheckAndInit` with `FromProtoResourceGroup`?